src/pages/getting-started/helpdesk-apps/index.mdx

```diff
@@ -0,0 +1,158 @@
+---
+weight: 6
+category: "getting-started"
+subcategory: "helpdesk-apps"
+title: "Building HelpDesk apps"
+tagline: "Learn all the ways to build upon HelpDesk."
+desc: "Do you want to create a HelpDesk app and automate some tasks or processes for your agents? Now, it's all possible. Implement convenient tweaks for your team, build integrations with other platforms, and create new products in the HelpDesk ecosystem."
+---
+
+# Introduction
+
+[Developer Console](https://developers.livechat.com/console/) allows you not only to create [LiveChat Apps](/getting-started/livechat-apps). It is also possible to develop integrations for <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a> and publish them on the Marketplace.
+
+Connect your HelpDesk to an external tool or service, extend HelpDesk user interface, or automate various tasks. Configure different functionalities, integrate with the LiveChat ecosystem, or create your own custom actions for agents in HelpDesk.
+
+# App capabilities
+
+LiveChat provides developers with multiple tools which allow your app to do many different things, for example:
+
+- Integrate with other products and services
+- Extend HelpDesk user interfaces
+- Create your own custom actions for agents in HelpDesk
+- Automate tasks with webhooks
+- Let users sign in to your service with Sign in with LiveChat
+
+If you plan to make your app public for everyone, check out our [monetization](/monetization/how-to-monetize-apps) and distribution tools.
+
+# Creating HelpDesk apps
+
+## Step-by-step guide
+
+If you want to create a simple HelpDesk app in the Developer Console, follow the instructions we’ve prepared. We'll use an app named Your Buddy as our example.
+
+### Step 1 Getting started
+
+To start, you need to check if you are signed in to the <Placeholder id="DEVELOPER_CONSOLE_URL" />. You can <a href="https://accounts.livechat.com/signup" target="_blank">sign up here</a>. However, your LiveChat or HelpDesk credentials should log you in automatically. 
+
+The first step in the Developer Console is to go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. The application's name can be changed later.
+
+<img
+  alt="Creating HelpDesk apps"
+  src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-creating new app.png"
+  width="1459px"
+  height="771px"
+/>
+
+Next, you need to decide which product you want to build your app for. In this tutorial guide, we will take care of the HelpDesk product. However, if you would like to check out how to build LiveChat apps, find more details [here](/getting-started/livechat-apps). The logo of the chosen product will be visible on the left side of your screen, under the **Building Blocks** tab.
```

Comment:
"However, if you would like to check out how to build LiveChat apps, find more details [here](/getting-started/livechat-apps)." 

It's a good idea to link to the LiveChat tutorial, but I would either put it somewhere else or shorten it, so that it doesn't distract the user.

> Next, you need to decide which product you want to build your app for. In this tutorial guide, we will take care of the HelpDesk product. Want to build for LiveChat instead? See the tutorial.